docs/Benodigde-tabellen.md

```diff
@@ -1,21 +1,49 @@
+| Functie_Adres                | LO GBA element 10.10 | Aanduiding van de functie van het adres.                                                                                                                   |
+| Gemeenten                    | Landelijke tabel 33  | Gemeenten                                                                                                                                                  |
+| Geslacht                     | LO GBA element 04.10 | Geeft aan wat het geslacht is van de persoon.                                                                                                              |
+| Gezagsverhouding             | Landelijke tabel 61  | Gezagsverhouding                                                                                                                                           |
+| Landen                       | Landelijke tabel 34  | Landen                                                                                                                                                     |
+| Naamgebruik                  | LO GBA element 61.10 | De manier waarop de geslachtsnaam van persoon en partner van persoon moet worden verwerkt in de manier waarop persoon wil worden aangesproken.             |
+| Nationaliteiten              | Landelijke tabel 32  | Nationaliteiten                                                                                                                                            |
+| Reden_Nationaliteit          | Landelijke tabel 37  | Reden opnemen - beeindigen Nationaliteit                                                                                                                   |
+| Reden_Opschorting_Bijhouding | LO GBA element 67.20 | Reden voor opschorting van de bijhouding.                                                                                                                  |
+| Soort_Verbintenis            | LO GBA element 15.10 | Soort verbintenis tussen de persoon en de partner die bij de burgerlijke stand is ingeschreven.                                                            |
+| Verblijfstitel               | Landelijke tabel 56  | Verblijfstitel                                                                                                                                             |
+
+## BRP-historie
+
+Voor de BRP-historie API moeten minimaal de volgende tabellen beschikbaar worden gemaakt:
+
+| tabelidentificatie        | bron                | omschrijving                                                           |
+|---------------------------|---------------------|------------------------------------------------------------------------|
+| Reden_Beeindigen_Huwelijk | Landelijke tabel 41 | Reden ontbinding nietigverklaring huwelijk geregistreerd partnerschap. |
```

Comment:
Ook de volgende tabellen zijn noodzakelijk:
* Gemeenten
* Landen
* Functie_Adres
* Aanduiding_Bij_Huisnummer
* Geslacht
* Soort_Verbintenis
* Verblijfstitel
* Reden_Nationaliteit
* Nationaliteiten
* Adellijke_Titel_Predicaat
*

Reply:
Ik denk dat het doel van dit document niet is om per API te zeggen welke tabellen gebruikt worden. Het doel is te zeggen welke tabellen minimaal geleverd moeten worden. Dan hoeft elke tabel maar één keer te worden genoemd.

Ik voeg de volgende tekst toe: "(bovenop wat voor BRP-Personen-Bevragen aan tabellen gevraagd is)".

Reply:
Prima!